src/pages/en/[...slug].astro

```diff
@@ -17,30 +15,32 @@ export async function getStaticPaths() {
   );
 
   return docEntries.map((entry, index) => {
-
     //removing .md extention AND "src/content/en" from filepath
-    const generatedSlug = entry.filePath?.replace("src/content/docs/en", "").replace(".md","");
+    const generatedSlug = entry.filePath
+      ?.replace("src/content/docs/en", "")
+      .replace(".md", "");
     const category = generatedSlug?.split("/")[1];
-    const isLearn =  entry.data.uid?.startsWith("help");
+    const isLearn = entry.data.uid?.startsWith("help");
     const isWebAPI = entry.data.uid?.startsWith("webapi");
 
     var TocPath;
-    if (isLearn) { TocPath = "learn" }
-    else if (isWebAPI) { TocPath = "api/reference/webapi" }
-    else { TocPath = category }
-    
+    if (isLearn) {
```

Comment:
As more and more content comes into the repo, this conditional logic will become impractical. Every root may contain a toc.yml today, should this this issue should be discussed.